tests/integration/targets/pipx/tasks/main.yml

```diff
@@ -8,17 +8,9 @@
     - name: Install pipx>=1.7.0
       pip:
         name: pipx>=1.7.0
-    - name: Set has_pipx170 fact true
-      ansible.builtin.set_fact:
-        has_pipx170: true
   rescue:
-    - name: Set has_pipx170 fact false
-      ansible.builtin.set_fact:
-        has_pipx170: false
-    - name: Install pipx (no version spec)
-      pip:
-        name: pipx
-
+    - name: Unable to install pipx 1.7.0 or higher, bailing out
+      ansible.builtin.meta: end_play
```

Comment:
One problem with using `rescue` is that in case installing pipx for another reason (like network problems etc.), the test will just be skipped without any indication (if you don't check out the logs). How about having `ansible.builtin.meta: ned_play` before the `pip` task with a `when` conditional that specifies for which platforms (and OS versions), or other conditions (Python version high enough?) to skip the test?

Reply:
Maybe I can pinpoint that with the Python version, yes - tox 1.7.0 requires Python 3.8 as minimum, so that might be the reason behind the failures that I saw before.

Reply:
Looks like it's working!